sql/core/src/main/java/org/apache/spark/sql/sources/v2/WriteSupport.java

```diff
@@ -38,15 +38,16 @@
    * If this method fails (by throwing an exception), the action will fail and no Spark job will be
    * submitted.
    *
-   * @param jobId A unique string for the writing job. It's possible that there are many writing
-   *              jobs running at the same time, and the returned {@link DataSourceWriter} can
-   *              use this job id to distinguish itself from other jobs.
+   * @param writeUUID A unique string for the writing job. It's possible that there are many writing
```

Comment:
what's the rationale behind this renaming?

Reply:
This is not the ID of the Spark job that is writing. I think the UUID name is more clear about what is actually passed, a unique string that identifies the write. There's also no need to make the string more complicated than a UUID since there are no guarantees about it.

Reply:
This is removed in the v2 API redesign.